releasenotes/notes/0.45/add-annotated-operations-de35a0d8d98eec23.yaml

```diff
@@ -55,6 +55,7 @@ features:
     As of now, the annotated operations can appear only in the top-level of a quantum
     circuit, that is they cannot appear inside of the recursively-defined ``definition``
     circuit. We are planning to remove this limitation later.
+features_synthesis:
```

Comment:
This is arguably a transpiler section, because the `HighLevelSynthesis` class is a transpiler pass. But synthesis also works too because it's about synthesizing `Annotated Operations`

Reply:
I know, I was a bit undecided, I think that I did label other `HighLevelSynthesis` notes as transpiler changes, so I can change this one too.